### PR TITLE
Add page on how to add schema source dir

### PR DESCRIPTION
**Pull request by paulbalaji**
#### Description

- small page on how to add schema source dir in gdk tools config
- update toc to have a `setup` section under blank project

#### Tests

- improbadoc serve

#### Documentation

- this is the docs

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/projects/blank/setup/schema.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+By default, the GDK finds and compiles schema found within `.schema/` folders of packages referenced in your project's `manifest.json`.
+
+To compile schema from directories outside of referenced packages, you must add your schema source directories to the GDK tools configuration.
+
+##### 1. Create a folder for schema
+
+First, create the folder in which you will be writing schema files.
+
+For reference, the FPS Starter Project contains a `schema/` folder at the root of its SpatialOS project.
```
Comment (jamiebrynes7) [inline docs/projects/blank/setup/schema.md]: It does not 😛

Comment (paulbalaji) [reply docs/projects/blank/setup/schema.md]: shh

File: docs/projects/blank/setup/schema.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+To add an entry, select the ➕ icon and enter the path to the schema directory you created earlier. **Note that the path must be relative to your Unity project directory.**
+
+> The GDK tools configuration window will display an error if the source directory does not exist.
+
+<img src="{{assetRoot}}assets/blank/schema/add-schema-source-after.png" style="margin: 0 auto; width: 50%; display: block;" />
+
+After adding this entry and ensuring that no errors are displayed, select **Save** and close the window.
+
+<%(#Expandable title="What should <code>GdkToolsConfiguration.json</code> look like when I'm done?")%>
```
Comment (zeroZshadow) [inline docs/projects/blank/setup/schema.md]: I don't think you've explained where this file is, or what it is for. So this feels very out of place.

File: docs/toc.md [hunk 3]
```diff
@@ -13,7 +13,10 @@
             - [5. Get playing]({{urlRoot}}/projects/fps/get-started/get-playing)
             - [6. View your game world]({{urlRoot}}/projects/fps/get-started/view-game-world)
         - [Health packs tutorial]({{urlRoot}}/projects/fps/tutorial)
-    - [Blank Starter Project]({{urlRoot}}/projects/blank/overview)
+    - Blank Starter Project
+        - [Overview]({{urlRoot}}/projects/blank/overview)
+        - Setup
```
Comment (zeroZshadow) [inline docs/toc.md]: Do we want this to have a sub-topic for just Add schema?

Comment (paulbalaji) [reply docs/toc.md]: I think it's fine for now - as the `add schema` bit will be covered by the blank project tutorial

(also a user _should_ know how to add schema if they take a look at the pickups tutorial)

**Pull request merged**